Makefile.buildx.mk

```diff
@@ -25,7 +25,7 @@ endif
 
 # default list of platforms for which multiarch image is built
 ifeq (${PLATFORMS}, )
-	export PLATFORMS="linux/amd64,linux/arm64"
+	export PLATFORMS="linux/amd64"
```

Comment:
Will arm be not built?

Reply:
Or is this about default and value will be overwritten from the workflow.. "Build and Push"

Reply:
This Makefile is no longer used for the workflows. We use the `docker/build-push-action` action which takes all such parameters as fields. This Makefile is maintained only for building & pushing the images locally. Missed to correct the platforms after I tested locally.